stock_move_location/models/stock_picking.py

```diff
@@ -0,0 +1,57 @@
+# Copyright Jacques-Etienne Baudoux 2016 Camptocamp
+# Copyright Iryna Vyshnevska 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import _, models
+from odoo.exceptions import UserError
+from itertools import groupby
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    def button_fillwithstock(self):
```

Comment:
@i-vyshnevska Why not using the existing wizard to compute qunatities ? IMHO this has duplicate code from wizard one. Maybe you could do something more smart that works in both contexts no ?

Reply:
@rousseldenis I have updated the code and reuse wizard features, but also I was needed to refactor wizard a little bit to fullfill our needs

Reply:
and I have no idea why travis fails